Packages/com.miniproject.core/Editor/Scripts/PackageWizard/EditorWindow/PackageWizard.cs

```diff
@@ -0,0 +1,211 @@
+	            HasSamples = false,//TODO Will need to add some support for this
+	            Version = "0.0.1",
+	            Description = GenerateDescription(m_AuthorName.text, m_AuthorDesc.text),
+	            AuthorInfo = new PackageData.Author
+	            {
+		            Name = "MiniProject",
+		            Email = "",
+		            Url = "https://github.com/navhayer1015/mini-project"
+	            }
+            };
+
+            var unityVersion = (PackageData.UnityVersion)_editorVersion.index;
+            Debug.Log($"Selected Editor Version {_editorVersion.index} | {unityVersion}");
+            _packageData.UnityVersions = new List<PackageData.UnityVersion> { unityVersion };
+
+            //Get Selected Tags
+            //----------------------------------------------------------//
+            _packageData.ExperienceTags = new List<PackageData.ExperienceTag>();
+            for (var i = 0; i < _tagToggles.Length; i++)
+            {
+	            var tagToggle = _tagToggles[i];
+	            
+	            if(tagToggle.value == false)
+		            continue;
+	            
+	            _packageData.ExperienceTags.Add((PackageData.ExperienceTag)i);
+            }
+
+            //Determine which Platforms were selected 
+            //----------------------------------------------------------//
+            _packageData.Platforms = new List<PackageData.Platform>();
+            var selectedPlatforms = (PackageData.Platform)_platformOptions.value;
+            foreach (Enum platform in Enum.GetValues(typeof(PackageData.Platform)))
+            {
+	            if (selectedPlatforms.HasFlag(platform) == false)
+		            continue;
+	            
+	            _packageData.Platforms.Add((PackageData.Platform)platform);
+            }
+            //----------------------------------------------------------//
+
+
+            var generator = new PackageGenerator(_packageData);
+            generator.OnProgressChanged += OnProgressChanged;
+            generator.Generate();
+        }
+
+        private void OnProgressChanged(object sender, ProgressEventArgs progress)
+        {
+            m_progressBar.value = progress.Progress * 100;
+            m_progressBar.title = progress.Info;
+        }
+
+		private void HandleGenerateButtonState()
+		{
+			bool textIsEmpty = m_packageNameInputField.text.Trim().Equals("");
+			m_GenerateButton.SetEnabled(!textIsEmpty);
+			SetWarning(textIsEmpty, R.ErrorMessages.EmptyNameError);
+
+		}
+
+		private void SetWarning(bool show, string message = ""){
+			m_warningLabel.text = message;
+			m_warningContainer.style.display = show ? DisplayStyle.Flex : DisplayStyle.None;
+		}
+
+		private void ClearTool()
+		{
+			m_progressBar.style.display = DisplayStyle.None;
+			m_packageNameInputField.SetValueWithoutNotify("");
+		}
+
+		private string GenerateDescription(string authorName, string description){
+			return string.Format("By: {0} \n\n{1}",authorName, description);
```

Comment:
Having an `[AuthorName]` variable (or By: [AuthorName]) in the readme template might be easier to understand